frontend/components/product-list/sections/ProductListChildrenSection.tsx

```diff
@@ -25,18 +25,18 @@ export function ProductListChildrenSection({
       childrenHeading,
       children: productListChildren,
    } = productList;
-   const [showingMore, setShowingMore] = React.useState(false);
+   const [isShowingMore, setShowingMore] = React.useState(false);
```

Comment:
If we're gonna change the name of the var, let's change the name of the setter to `setIsShowingMore` too just to be consistent

Reply:
I'll apply the rename in one of my next PR